addon/helpers/dynamic-image.js

```diff
@@ -0,0 +1,32 @@
+import { htmlSafe } from '@ember/template';
+export default function dynamicImage([imageSrc], hash) {
```

Comment:
We should probably leave a comment that this is deprecated and should be removed once Sponsors is updated in editor and core.

Reply:
✅